### PR TITLE
Comment screen - UI and Load comment

### PR DESCRIPTION
**Pull request by at-huyvo2**
- UI comment
- Load all comment from API
- Handle keyboard when typing

### CODE REVIEW

File: MyApp/View/Base/BaseViewController.swift [hunk 1]
```diff
@@ -10,6 +10,10 @@ import UIKit
 import MVVM
 
 class BaseViewController: UIViewController, View {
+
```
Comment (at-daiho) [inline MyApp/View/Base/BaseViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

File: MyApp/View/Controller/CommentViewController/CommentViewController.swift [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    override func setupUI() {
+        tabBarController?.tabBar.isTranslucent = true
+        commentTextField.delegate = self
+        addButtonWidthContraint.constant = Config.hideKeyboardAddButtonWidth
+        configTableView()
+        configKeyboard()
+    }
+
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        commentTextField.resignFirstResponder()
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/CommentViewController/CommentViewController.swift]: Thiếu `supper.touchesBegan`

File: MyApp/View/Controller/CommentViewController/CommentViewController.swift [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    private func configTableView() {
+        let nib = UINib(nibName: Config.identifierTableCell, bundle: Bundle.main)
+        tableView.register(nib, forCellReuseIdentifier: Config.identifierTableCell)
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.keyboardDismissMode = .onDrag
+    }
+
+    private func addComment() {
+        commentTextField.resignFirstResponder()
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/CommentViewController/CommentViewController.swift]: Hàm này để thêm comment hay là ẩn bàn phím

File: MyApp/View/Controller/DetailViewController/Cell/CommentCell/CommentCell.swift [hunk 4]
```diff
@@ -27,21 +27,20 @@ final class CommentCell: UITableViewCell, View {
...
         let path = viewModel.iconFarm == 0 ? Config.defaultAvatar : viewModel.avatarPath
         avatarImageView.setImage(path: path)
     }
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        avatarImageView.layer.cornerRadius = frame.size.width / Config.ratio
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        avatarImageView.layer.cornerRadius = avatarImageView.frame.height / Config.ratio
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/DetailViewController/Cell/CommentCell/CommentCell.swift]: Tại sao lại bỏ ở layoutSubviews

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**